lib/streamlit/DeltaGenerator.py

```diff
@@ -1876,6 +1881,39 @@ def slider(
             current_value = current_value[0] if single_value else current_value
         return current_value if single_value else tuple(current_value)
 
+    @_with_element
+    def file_uploader(self, element, label, type=['*'], key=None):
+
+        """Display a file uploader widget.
+
+        Parameters
+        ----------
+        label : str or None
+            A short label explaining to the user what this file uploader is for.
+        type : string[] or None
+            Array of allowed extensions. (['.png', '.jpg'])
+            Default value: ['*']
```

Comment:
This means `type` is optional, right? (Which I think is the right approach)

However, if I don't pass a type, and then I try to upload a file, I get an error.

Steps to repro:
1) `st.file_uploader("Upload a file")
2) run the script
3) upload an SVG file, for example

**Expected:** it should just work
**Result:** this error shows up `image/svg+xml files are not allowed`


When  like "image/svg+xml files are not allowed"